lib/src/types.ts

```diff
@@ -235,6 +257,7 @@ export type YAxisInputProps<
   domain?: YAxisDomain;
   linePathEffect?: DashPathEffectComponent;
   enableRescaling?: boolean;
+  tickImages?: TickImage[];
```

Comment:
If we go the route with `formatYLabel` / `renderYLabel`, I think we get rid of `tickImages`

Reply:
done